openpype/tools/utils/widgets.py

```diff
@@ -586,3 +592,204 @@ def set_orientation(self, orientation):
         self._orientation = orientation
 
         self._set_size(self._size)
+
+
+class PlaceholderHistoryLineEdit(QtWidgets.QComboBox):
+    def __init__(self, placeholder_text="...", parent=None):
+        super(PlaceholderHistoryLineEdit, self).__init__(parent=parent)
+        self.setEditable(True)
+        self.setPlaceholderText(placeholder_text)
+
+
+class ToolbarButton(QtWidgets.QPushButton):
+    def __init__(self, *args, **kwargs):
+        super(ToolbarButton, self).__init__(*args, **kwargs)
+        self.setFixedWidth(35)
```

Comment:
It had a reason the width and height are same. Also using fixed sizes cause issues when someone has high resolution and enabled scaling.

Reply:
I can't see where the previous refresh button had it's width and height set?
Unless you have another suggestion, I will use `QtWidgets.QToolButton` instead as that maintains a square ratio?